backend/models/helper/model.js

```diff
@@ -492,11 +492,11 @@ function searchTree(account, model, branch, rev, searchString, username) {
 			name: new RegExp(searchString, "i")
 		};
 
-		return Scene.find({account, model}, filter, { name: 1 }).then(objs => {
+		return findScenes(account, model, filter, { name: 1 }).then(objs => {
```

Comment:
We're finding nodes within a scene here. I think it'd be helpful if we call this findNode or something.

Also this is super high coupling and once again we're just exposing a .find function.

We should probably encapsulate the logic within the Scene file and have this takein the revisionId/branch, object type, and the search string